lib/bolt_server/pe/pe_pal.rb

```diff
@@ -0,0 +1,135 @@
+require 'bolt/pal'
+require 'bolt/util'
+
+module PEBolt
```

Comment:
We try fairly hard to put modules and files in predictable paths.

so `BoltServer::PE::PE_PAL` or something.